### PR TITLE
Update code style to Black standards

### LINKED ISSUE #99: Ensure code adheres to coding standards.
### Expected Behavior

As decided on slack channel, we are following PEP8 style for python and Google code formating for C++. This was pushed in #98 

### Current Behavior

The current code does not adhere to these standards and fails. 

###  How to make it work

- If you type `pycodestyle `in the root directory, you will get list of all errors that needs to fixed. 
- To fix CPP files, make sure that you run clang-format. https://leimao.github.io/blog/Clang-Format-Quick-Tutorial/ might come in handy. 
Following commands needs to fix this,
```
clang-format -i src/bindings/PyDP/*/*.cpp
clang-format -i src/bindings/PyDP/*/*.hpp
```

### PR DESCRIPTION
**Pull request by alejandrosame**
## Description

In this PR Python code has been formatted using the code formatter [Black](https://github.com/psf/black). Accordingly, the pycodestyle configuration has been updated to allow lines up to 88 characters. The same configuration has been applied to clang-format configuration file, and the C++ code has been reformatted to comply with it.

Fixes #99

## Type of change

Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Did you run `make test-all`?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] New Unit tests added
- [ ] Unit tests pass locally with my changes

### CODE REVIEW

**Comment by alejandrosame** [comment]
Hi again!

It took some separate tries, but now the changes and the tests are congruent with Black code style. 

I had to take out the code style checks to a separate workflow file because Black needs python3.6+ to run (and also, the style check does not need to be executed only needs to be run once, regardless of Python version to be tested).

File: src/bindings/PyDP/base/logging.cpp [hunk 1]
```diff
@@ -18,9 +18,7 @@ class Logging_helper {
   }
   int get_vlog_level() { return dpbase::get_vlog_level(); }
 
-  std::__cxx11::string get_log_directory() {
-    return dpbase::get_log_directory();
-  }
+  std::__cxx11::string get_log_directory() { return dpbase::get_log_directory(); }
```
Comment (chinmayshah99) [inline src/bindings/PyDP/base/logging.cpp]: Do we want the function definition in the same-line?

Comment (alejandrosame) [reply src/bindings/PyDP/base/logging.cpp]: This is consistent with the Differential Privacy codebase (that is, when reading Google code, we will find this formatting). Once again, I don't have any strong argument against it (besides personal preference).

**Comment by chinmayshah99** [comment]
Hi @alejandrosame can you resolve the merge conflicts?

File: src/bindings/PyDP/base/logging.cpp (hunk 1 above)
Comment (alejandrosame) [reply src/bindings/PyDP/base/logging.cpp]: Now clang does not format short functions in one line

**Review by chinmayshah99 — APPROVED** [review]

**Pull request merged**